docs/getting-started.txt

```diff
@@ -1,300 +1,338 @@
-#################
- Getting Started
-#################
+#########################
+ Quickstart for ML Users
+#########################
 
-This section walks through the steps to run examples in Determined from zero. You should be able to:
+This quickstart walks you through the basic steps to install and run Determined, train on the
+provided model data, and visualize training experiment results in your browser.
 
-#. Get a light touch with Determined concepts and features;
-#. Install Determined;
-#. Get familiar with the WebUI and the CLI;
-#. Train an example model with a single GPU;
-#. Train an example model with distributed training;
-#. Train an example model with hyperparameter tuning.
+Three model data and code examples use the MNIST dataset to demonstrate:
 
-*********************
- System Requirements
-*********************
+-  training on a local, single CPU with or without a GPU.
+-  how to run a distributed training job on multiple GPUs to improve performance.
+-  how to increase model accuracy using hyperparameter tuning.
```

Comment:
```suggestion
-  how to use hyperparameter tuning.
```

Reply:
OK. Good, 'use' assumes the reader already knows what hyperparameter tuning is for.